src/components/ReactTags.js

```diff
@@ -439,6 +442,7 @@ class ReactTags extends Component {
           name={inputName}
           id={inputId}
           maxLength={maxLength}
+          autoComplete={inputFieldAutocomplete}
```

Comment:
@atiqueansari1987 lets not make it as a prop. instead lets pass it as a rest prop ? in that way these fields need not have a prop, whatever the parent passes in inputProps will be added here as rest props. what do you think ?

Reply:
Yes, we can have something like this.

```
<input
          ref={(input) => {
            this.textInput = input;
          }}
          className={this.state.classNames.tagInputField}
          type="text"
          placeholder={placeholder}
          aria-label={placeholder}
          onFocus={this.handleFocus}
          onBlur={this.handleBlur}
          onChange={this.handleChange}
          onKeyDown={this.handleKeyDown}
          onPaste={this.handlePaste}
          name={inputName}
          id={inputId}
          value={this.props.inputValue}
          {...inputAttributes}
/>
```

Shall I remove input related props and introduce inputAttributes which will accept an object.

Reply:
yup pls remove input related props and rename `inputAttributed` to `inputProps`.